internal/assets/dockerfiles/java-war-tomcat/m2kdfdetect.sh

```diff
@@ -0,0 +1,22 @@
+#   Copyright IBM Corporation 2020
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# Takes as input the source directory and returns error if it is not fit
+main() {
+    [ "$#" -gt 1 ] && echo 'multiple WAR files. exiting' && exit 1
```

Comment:
If we have multiple wars let's rake the first war till we start supporting multiple options with an array

Reply:
so should the logic for jboss and liberty be changed as well?

Reply:
yes. But we can merge this one, and do it all together in another PR.